platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationFilesPoller.java

```diff
@@ -80,16 +72,7 @@ public void init() throws IOException {
                 this.getClass()
                         .getName());
 
-        try {
-            LOGGER.debug("Registering path [{}] with Watch Service.",
-                    configurationDirectoryPath.toString());
-            configurationDirectoryPath.register(watchService, ENTRY_CREATE);
-        } catch (IOException e) {
-            LOGGER.error("Unable to register path [{}] with Watch Service",
-                    configurationDirectoryPath.toString(),
-                    e);
-            throw e;
-        }
+        watchService.addListener(new ListenerImpl());
```

Comment:
❗️ Does this get the polling started or does the polling start the first time `checkAndNotify()` is called on L90? I'm asking because we need to make sure we don't process the same configuration file twice, once because `processExistingConfigurationFiles()` is called in the `register()` method and once because `ListenerImpl` gets notified.